returnn/datasets/hdf.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, files=None, use_cache_manager=False, **kwargs):
     self.data_dtype = {}  # type: typing.Dict[str,str]
     self.data_sparse = {}  # type: typing.Dict[str,bool]
     self._num_codesteps = None  # type: typing.Optional[typing.List[int]]  # accumulated sequence length per target
+    self._tag_to_real_idx_map = None  # type: typing.Optional[typing.Dict[str,int]]
```

Comment:
Why do you introduce that? You already have `_tag_idx` which is exactly the same?